_config.yml

```
@@ -7,9 +7,15 @@ author: The Python in Chemistry Community
 
 # Force re-execution of notebooks on each build.
 # See https://jupyterbook.org/content/execute.html
+only_build_toc_files: true
 execute:
   execute_notebooks: force
```

```suggestion
  execute_notebooks: force
  allow_errors: true
```

This will help the debugging exercise thing.